static-site/gatsby-node.js

```diff
@@ -200,6 +200,12 @@ exports.createPages = ({graphql, actions}) => {
           component: path.resolve("src/sections/individual-group-page.jsx")
         });
 
+        createPage({
+          path: "/groups/:groupName/settings",
+          matchPath: "/groups/:groupName/settings/*",
+          component: path.resolve("src/sections/edit-group-settings-page.jsx")
+        });
```

Comment:
Renamed in 55a7d4a8868e215a06fd56f4528dd00c556b4fd0. 

(This tripped me up for a bit because Gatsby kept trying to find the old file...It finally worked after I deleted my local `static-site/.cache/` directory)